public/pages/CreateDetector/components/ConfigureAlerts/components/AlertCondition/AlertConditionPanel.tsx

```diff
@@ -537,6 +559,9 @@ export default class AlertConditionPanel extends Component<
 
         <NotificationForm
           action={alertCondition.actions[0]}
+          context={{
+            ctx: context,
+          }}
```

Comment:
This doesn't seem to match the expected type `TriggerContext`. It doesn't have `ctx` as the expected field.

Reply:
Done